src/components/Settings/SettingsTab.tsx

```diff
@@ -37,6 +37,7 @@ export const SettingsTab = ({
   const [updateTS, setUpdateTS] = useState(0);
   const [vanity, setVanity] = useState<string | undefined>(undefined);
   const [password, setPassword] = useState<string | undefined>(undefined);
+  const [isChatDisabled, setisChatDisabled] = useState(true);
```

Comment:
shouldn't we default this to false?

Also I think it should be setIsChatDisabled